packager/debian/postinst

```diff
@@ -28,6 +28,12 @@ case "$1" in
 
     # scale
     ${CLI} scale web=1 || true
+    
+    #restart the service
```

Comment:
I don't want to be picky here, but all the other comments are in the format `# <content>`. Why make this one `#<content>`?